cloud/parameterstore/fakeparameter/fake_parameter_test.go

```diff
@@ -0,0 +1,9 @@
+package fakeparameter
+
+import (
+	"github.com/evergreen-ci/evergreen/testutil"
+)
+
+func init() {
+	testutil.Setup()
+}
```

Comment:
Is this meant to only have init?

Reply:
It doesn't really matter where the init goes, so I can move it into the other test file to avoid the empty file.